### DISCUSSION TITLE
ARRAY and STRUCT support

### DISCUSSION THREAD

**Discussion by vadrianc** [COVESA/dlt-daemon#565] [Q&A]
As defined in the "Specification of Diagnostic Log and Trace AUTOSAR Release 4.2.2" and "PRS 1.4.0", the DLT protocol allows for type STRUCT and type ARRAY verbose arguments to be contained in the payload.

 Are there any plans to implement support for these argument types?

**Comment by ssugiura** [comment]
@vadrianc Thanks for pointing out. We are aware of those kind of problem. As AUTOSAR has newer release which some protocols are different from ours, we are planning to update accordingly.
